src/moov/trak/mdia/minf/stbl/stsd/chnl.rs

```diff
@@ -0,0 +1,873 @@
+use crate::*;
+
+// ISO/IEC 23091-3 Audio Channel Position
+#[derive(Debug, Clone, Copy, PartialEq, Eq)]
+#[cfg_attr(feature = "serde", derive(serde::Serialize, serde::Deserialize))]
+#[repr(u8)]
+pub enum AudioChannelPosition {
+    FrontLeft = 0,
+    FrontRight = 1,
+    FrontCenter = 2,
+    Lfe1 = 3,
+    LeftSurround = 4,
+    RightSurround = 5,
+    FrontLeftOfCenter = 6,
+    FrontRightOfCenter = 7,
+    RearLeft = 8,
+    RearRight = 9,
+    RearCenter = 10,
+    SurroundLeft = 11,
+    SurroundRight = 12,
+    SideLeft = 13,
+    SideRight = 14,
+    FrontLeftWide = 15,
+    FrontRightWide = 16,
+    TopFrontLeft = 17,
+    TopFrontRight = 18,
+    TopFrontCenter = 19,
+    TopRearLeft = 20,
+    TopRearRight = 21,
+    TopRearCenter = 22,
+    TopSideLeft = 23,
+    TopSideRight = 24,
+    TopCenter = 25,
+    Lfe2 = 26,
+    BottomFrontLeft = 27,
+    BottomFrontRight = 28,
+    BottomFrontCenter = 29,
+    TopSurroundLeft = 30,
+    TopSurroundRight = 31,
+    // 32-35: reserved
+    Lfe3 = 36,
+    Leos = 37,
+    Reos = 38,
+    Hwbcal = 39,
+    Hwbcar = 40,
+    Lbs = 41,
+    Rbs = 42,
+    // 45-125: reserved
```

Comment:
```suggestion
    /// Left ear (Lear).
    ///
    /// From ISO/IEC 23091-3:2018/Amd. 1:2022(E):
    /// > Audio signals associated with this loudspeaker position are intended for stereo headphone playback only. They can be
    /// > unsuitable for loudspeaker playback. This loudspeaker position is applicable for binaural signals but it shall not be applied
    /// > for traditional stereo signals.
    LeftEar = 43,
    /// Right ear (Rear).
    ///
    /// From ISO/IEC 23091-3:2018/Amd. 1:2022(E):
    /// > Audio signals associated with this loudspeaker position are intended for stereo headphone playback only. They can be
    /// > unsuitable for loudspeaker playback. This loudspeaker position is applicable for binaural signals but it shall not be applied
    /// > for traditional stereo signals.
    RightEar = 44,
    // 45-125: reserved
```
(will probably need reformatting).

Reply:
Fixed these in the PR against your branch. Can you verify?